README.rst

```diff
@@ -85,8 +85,6 @@ Quick-start
 .. code::
 
    git clone https://github.com/USERNAME/DART.git
```

Comment:
i like the other places in our instructions where it doesn't start by telling you to fork, but uses NCAR instead of username, and then below this line can say:

If you have forked the DART repository, replace ``NCAR`` with your Github username.

if you want to keep the fork instructions, then steps 3,4,5 have to say how to sync your fork with the NCAR repo and how to issue a pull request to the NCAR repo.  right now it's telling users to make a pull request to their own fork which doesn't inform us of anything.  i'd prefer to replace all these instructions with the lines from guide/downloading-dart.rst.

people who know git will understand how to make and manage their own fork.  users not well versed in git will thank you to make this as simple as possible.  (this includes me.)

Reply:
added a quick-start for developers with forking, adding the upstream and pull requests.  Is this clear enough?

Reply:
i think splitting it into regular user and developer is great, and i think the new instructions are more clear now.  so yes.